src/main/scala/io/archivesunleashed/df/package.scala

```diff
@@ -59,6 +60,38 @@ package object df {
 
   val DetectMimeTypeTikaDF = udf((io.archivesunleashed.matchbox.DetectMimeTypeTika.apply(_: Array[Byte])))
 
+  val hasHTTPStatus = udf((statusCode: String, statusCodes: Seq[String]) => statusCodes.contains(statusCode))
+
+  val hasMIMETypes = udf((mimeType: String, mimeTypes: Seq[String]) => mimeTypes.contains(mimeType))
+
+  val hasMIMETypesTika = udf((mimeTypeTika: String, mimeTypes: Seq[String]) => mimeTypes.contains(mimeTypeTika))
```

Comment:
Here, I think if we follow the convention of other UDF's, it should be "mimeTypesTika: Seq[String]".
I had by mistake used the wrong convention for MIMETypesTika while writing serializable APIs previously.

Reply:
I meant, currently it is "mimeTypes: Seq[String]" but it should be "mimeTypesTika: Seq[String]" in the udf.